scripts/create_pelorus_operator

```diff
@@ -32,7 +32,7 @@ DEFAULT_PELORUS_OPERATOR_DIR="${SCRIPT_DIR}/../pelorus-operator/"
 DEFAULT_PELORUS_OPERATOR_PATCHES_DIR="${SCRIPT_DIR}/pelorus-operator-patches/"
 
 # Needed for symlink inside pelorus-operator folder
-DEFAULT_OPERATOR_HELM_CHART_SUBDIR="helm-charts/pelorus"
+# DEFAULT_OPERATOR_HELM_CHART_SUBDIR="helm-charts/pelorus"
```

Comment:
why comment this line?
(can we delete it?)

Reply:
It applies to lines 103->113, so we would need to remove all of that, which is marked TBD.

I have commented this out because shellcheck was failing on my local box complaining that the variable is unused.

Reply:
removed commented out block in the newest version of this PR.